py/desitarget/sv1/sv1_cuts.py

```diff
@@ -1734,10 +1729,10 @@ def set_target_bits(photsys_north, photsys_south, obs_rflux,
     desi_target |= qso * desi_mask.QSO
 
     # ADM add the per-bit information in the south for LRGs...
-    desi_target |= lrginit_s_4 * desi_mask.LRG_INIT_4PASS_SOUTH
-    desi_target |= lrgsup_s_4 * desi_mask.LRG_SUPER_4PASS_SOUTH
-    desi_target |= lrginit_s_8 * desi_mask.LRG_INIT_8PASS_SOUTH
-    desi_target |= lrgsup_s_8 * desi_mask.LRG_SUPER_8PASS_SOUTH
+    desi_target |= lrg_opt_s * desi_mask.LRG_OPT_SOUTH
+    desi_target |= lrg_ir_s * desi_mask.LRG_IR_SOUTH
+    desi_target |= lrg_sv_opt_s * desi_mask.LRG_OSV_PT_SOUTH
```

Comment:
In the `py/desitarget/sv1/data/sv1_targetmask.yaml` file, you called this bit `LRG_SV_OPT_SOUTH` (not `LRG_OSV_PT_SOUTH`). I'm not sure why this isn't failing in your local tests, but unit tests caught it.

Reply:
Thanks. It's fixed.